package.json

```diff
@@ -77,12 +78,14 @@
     "streamify-string": "^1.0.1",
     "ts-jest": "^29.0.0",
     "typedoc": "^0.21.0",
-    "@strictsoftware/typedoc-plugin-monorepo": "^0.4.2",
     "typescript": "^4.0.5"
   },
   "pre-commit": [
     "build",
     "lint",
     "test"
-  ]
+  ],
+  "dependencies": {
+    "@lerna/project": "^6.4.1"
```

Comment:
Why is this needed?

Reply:
Ok, weird that it's not needed in other projects using tsdoc.
Anyways, should be fixable in the future by upgrading to the latest tsdoc.

In any case, could you move this to a devDep?

Reply:
Done!